### PR TITLE
Add photo link to users table

### PR DESCRIPTION
**Pull request by denvaar**
- Update auth process to show bad request error

### CODE REVIEW

File: lib/show_n_tell/github/authenticator.ex [hunk 1]
```diff
@@ -24,6 +24,7 @@ defmodule ShowNTell.Github.Authenticator do
     {:fail, body}
   end
 
+  defp check_github_org({:ok, %{status: 401} = resp}), do: {:fail, %{"error" => Map.get(resp.body, "message")}}
```
Comment (alex-kovshovik) [inline lib/show_n_tell/github/authenticator.ex:27]: Cool - this handles the 401 - unauthorized, un addition to the "catch-all" error handler below.

File: lib/show_n_tell_web/controllers/auth_controller.ex [hunk 2]
```diff
@@ -11,6 +11,18 @@ defmodule ShowNTellWeb.AuthController do
     |> render_result(conn)
   end
 
+  def show(conn, _params) do
```
Comment (alex-kovshovik) [inline lib/show_n_tell_web/controllers/auth_controller.ex:14]: Nice - showing authorization!

File: lib/show_n_tell_web/router.ex [hunk 3]
```diff
@@ -8,7 +8,7 @@ defmodule ShowNTellWeb.Router do
   scope "/api", ShowNTellWeb do
     pipe_through :api
 
-    post "/authenticate", AuthController, :create
+    resources "/authenticate", AuthController, singleton: true, only: [:create, :show]
```
Comment (alex-kovshovik) [inline lib/show_n_tell_web/router.ex:11]: 👍

File: lib/show_n_tell_web/controllers/auth_controller.ex [hunk 4]
```diff
@@ -11,6 +11,18 @@ defmodule ShowNTellWeb.AuthController do
...
+    [authorization_header] =
+      conn
+      |> get_req_header("authorization")
+
+    "bearer " <> token = authorization_header
```
Comment (alex-kovshovik) [inline lib/show_n_tell_web/controllers/auth_controller.ex:19]: Had to lookup `<>` operator again. In Ruby it is recommended to use string interpolation for performance reasons:

`auth_header = "bearer #{token}"`

In Elixir - I'm not sure.

**Review by alex-kovshovik — APPROVED** [review]
Looks great!

File: lib/show_n_tell_web/controllers/auth_controller.ex (hunk 4 above)
Comment (denvaar) [reply lib/show_n_tell_web/controllers/auth_controller.ex:19]: @alex-kovshovik I am using `<>` for pattern matching. I needed to extract only the token from the string "bearer 232rqf34fq3f34"

Comment (denvaar) [reply lib/show_n_tell_web/controllers/auth_controller.ex:19]: I don't think you can pattern match with string interpolation: 
```
iex(1)> header = "bearer asdfa2423f2"
iex(2)> "header #{token}" = header
** (MatchError) no match of right hand side value: "bearer asdfa2423f2"
```

Comment (alex-kovshovik) [reply lib/show_n_tell_web/controllers/auth_controller.ex:19]: Ah, I see, I didn't get it then :) I forgot you could do that in Elixir - this is awesome!

File: lib/show_n_tell_web/controllers/auth_controller.ex (hunk 2 above)
Comment (denvaar) [reply lib/show_n_tell_web/controllers/auth_controller.ex:14]: Yes, this controller action will allow us to retrieve a user based on an access token

**Pull request merged**